tests/legacy-cli/e2e/utils/process.ts

```diff
@@ -96,39 +96,47 @@ function _exec(options: ExecOptions, cmd: string, args: string[]): Promise<Proce
   _processes.push(childProcess);
 
   // Create the error here so the stack shows who called this function.
```

Comment:
Seems like this wasn't actually done before? Creating the `error` with the stack trace at this point is the primary change. Otherwise the stack trace is always just the child_process exit/error stack which is useless.